lib/common.js

```diff
@@ -95,12 +103,15 @@ exports.doDeleteApiCall = function (host, uri, payload, errorMsg, cb) {
     return cb(undefined, remoteData);
   });
 };
+// END api calls
```

Comment:
Try introducing comments on blocks of similar functions in `common.js` as they can be broken down to smaller files in the future.